<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerDialog</class>
 <widget class="QDialog" name="ComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>338</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_addComputerTitle">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Add &lt;/span&gt;&lt;span style=&quot; font-size:18pt; color:#377BCE;&quot;&gt;Computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_ComputerName">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Computer Name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="buddy">
        <cstring>inputComputerName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_NameError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="inputComputerName"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_ComputerType">
       <property name="text">
        <string>Type</string>
       </property>
       <property name="buddy">
        <cstring>inputComputerType</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_TypeError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="inputComputerType"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_wasComputerBuilt">
     <property name="text">
      <string>Was the computer built?</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="radioButtonWasBuilt">
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QRadioButton" name="radioButtonNotBuilt">
     <property name="text">
      <string>No</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Build year?</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QDateEdit" name="computerBuildYear">
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1752</year>
       <month>9</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_AddTheComputer">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_CancelAdd">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputComputerName</tabstop>
  <tabstop>inputComputerType</tabstop>
  <tabstop>radioButtonWasBuilt</tabstop>
  <tabstop>radioButtonNotBuilt</tabstop>
  <tabstop>computerBuildYear</tabstop>
  <tabstop>pushButton_AddTheComputer</tabstop>
  <tabstop>pushButton_CancelAdd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
